ProcessMaker/Providers/RouteServiceProvider.php

```diff
@@ -89,7 +89,7 @@ protected function mapApiRoutes()
     {
         Route::middleware('api')
             ->group(base_path('routes/api.php'));
```

Comment:
Hey @eiresendez I noticed that the etag middleware is missing from the `Route::middleware('api')`, `Route::middleware('engine')` and `Route::middleware('web')` groups. 
Adding it will really help with caching, boost performance globally and we could close the PR of eTag in Cases/FOUR-20920.

Reply:
Hi @estebangallego 
For API routes is covered in this line: `Route::pushMiddlewareToGroup('api', HandleEtag::class);`.
For engines, I just pushed a commit for that 👍 .
For web routes: I’m not entirely sure applying the middleware, so this might be out of scope..